examples/graphql-api/app/utils/index.ts

```diff
@@ -0,0 +1,27 @@
+import { GRAPHQL_API } from "~/config";
```

Comment:
Maybe `GRAPHQL_API` should come from the environment variables.

Reply:
Sure thing! I'm not sure how that will work with the Code Sandbox link, any idea there or would the user have to duplicate the file in there?